cluster-provision/k8s/1.30/provision.sh

```diff
@@ -58,8 +60,16 @@ dnf install -y container-selinux
 
 dnf install -y libseccomp-devel
 
-dnf install -y centos-release-nfv-openvswitch
-dnf install -y openvswitch2.16
+#openvswitch for s390x is not available from the centos default repos.
+if [ "$ARCH" == "s390x" ]; then
+  dnf install -y https://kojipkgs.fedoraproject.org//packages/openvswitch/2.16.0/2.fc36/s390x/openvswitch-2.16.0-2.fc36.s390x.rpm
```

Comment:
Would there be much needed to get this in the default repos?

Reply:
I'm not sure. But, I will try finding how to do this and whom to contact, etc. If you know whom to reach, please let me know.

Reply:
I've asked the question in fedora-devel group https://matrix.to/#/!DMptnKvPTdgoULzUTV:fedoraproject.org/$rbqZqn1e4e5pIJ_UAnjr5xnJJor4c5M3RDFF-fMU5sk?via=fedoraproject.org&via=fedora.im&via=matrix.org

Reply:
@brianmcarey : I checked with community, basically there are no builds created by [CentOS nfv SIG](https://sigs.centos.org/nfv/) community for s390x. So, better I build from [sources](https://git.centos.org/rpms/openvswitch/branches?branchname=c8s-sig-nfv-openvswitch-2.16) (or better from [srpm](https://mirror.stream.centos.org/SIGs/9-stream/nfv/source/openvswitch-2/Packages/o/)), which has spec file to build an rpm.

Reply:
The current place from where I'm downloading is also from authentic user releng (it is fedora's release engineering), so I'm thinking may be as a follow-up PR I can fix building it from sources, if you are ok. Otherwise also let me know I will try building rpm from sources. I still need to figure out what exact commands/flags need to pass to rpm-build/moc, to be consistent with that of x86.

Reply:
Yes I followed the conversation over on the fedora matrix - this should be ok for now.